src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -124,7 +124,48 @@ export const QueryResultPane = () => {
         qr.QueryResultWebviewState,
         qr.QueryResultReducers
     >();
-    var metadata = state?.state;
+    const metadata = state?.state;
+
+    // lifecycle logging right after context consumption
+    useEffect(() => {
+        console.debug("QueryResultPane mounted", {
```

Comment:
If these logs are important enough, we should think about adding a logger in our webview framework that will write these logs to a file. Will be helpful when debugging client issue

Reply:
Created an issue here for tracking: https://github.com/microsoft/vscode-mssql/issues/18540